webapp/security/helpers.py

```diff
@@ -21,16 +23,23 @@ def get_summarized_status(
     for package in cve["packages"]:
         package_count += len(package["statuses"])
```

Comment:
I see `package_count` being used in `total_fixable` below, I think we need to tweak this so we filter out unselected releases.